homeassistant/config_entries.py

```diff
@@ -1468,6 +1468,28 @@ async def _old_conf_migrator(old_config: dict[str, Any]) -> dict[str, Any]:
     return {"entries": old_config}
 
 
+@callback
+def _async_abort_entries_match(
+    other_entries: list[ConfigEntry], match_dict: dict[str, Any] | None = None
+) -> None:
+    """Abort if current entries match all data.
+
+    Requires `already_configured` in strings.json in user visible flows.
+    """
+    if match_dict is None:
+        match_dict = {}  # Match any entry
```

Comment:
Side note: This is cryptic I think. If we just want to abort, we should just abort. Am I missing how this can be used?

Reply:
If i follow the code correctly, we don't want to "just abort".
We still want to see if there is an existing entry.

It seems to be a shorthand for "single instance allowed"

Reply:
We could change it to something like this:

```python
    if not match_dict:
        # Match any entry
        if other_entries:
            raise data_entry_flow.AbortFlow("already_configured")
        return
```

Reply:
Right. I guess it saves one line.